modules/apps/headless/headless-delivery/headless-delivery-test/src/testIntegration/java/com/liferay/headless/delivery/resource/v1_0/test/StructuredContentResourceTest.java

```diff
@@ -1035,6 +1094,70 @@ private StructuredContentResource _buildStructureContentResource(
 		).build();
 	}
 
+	private JSONArray _createBatchBody(long structureId) {
+		JSONObject structuredContentJSONObject = JSONUtil.put(
```

Comment:
Could we use this other method: https://github.com/liferay/liferay-portal/blob/fa86ff541a4a3ce707662980f49e51d59f424623/modules/apps/headless/headless-delivery/headless-delivery-client/src/main/java/com/liferay/headless/delivery/client/dto/v1_0/StructuredContent.java#L816 to transform the StructuredContent into a JSON String?